config/clients/dotnet/template/Client/Client.mustache

```diff
@@ -16,7 +16,6 @@ public class {{appShortName}}Client : IDisposable {
     private string CLIENT_METHOD_HEADER = "{{clientMethodHeader}}";
 
     private readonly int DEFAULT_MAX_METHOD_PARALLEL_REQS = {{clientMaxMethodParallelRequests}};
-    private readonly int DEFAULT_WAIT_TIME_BETWEEN_CHUNKS_IN_MS = {{clientDefaultWaitTimeBetweenChunksInMs}};
```

Comment:
Just to double verify ... this change is for .NET

Reply:
Yep - it was unused.

We dropped this field now that it has been removed from the JS SDK. The OpenFgaClient in the .NET and Go SDKs used proper parallelization from the get go.